pyvista/utilities/reader.py

```diff
@@ -1478,29 +1483,97 @@ def set_active_time_point(self, time_point):  # noqa: D102
         self.set_active_time_value(self.time_values[time_point])
 
 
+class DICOMReader(BaseReader):
+    """DICOM Reader for reading ``dcm`` files.
+
+    This reader reads a single file or a path containing a several ``dcm``
+    files (DICOM stack).
+
+    Parameters
+    ----------
+    path : str
+        Path to the single DICOM (``dcm``) file to be opened or the directory
+        containing a stack of DICOM files.
+
+    Raises
+    ------
+    FileNotFoundError
+        If ``path`` is not a DICOM file (``dcm``) or a directory containing
+        DICOM files.
+
+    Examples
+    --------
+    >>> import pyvista
+    >>> from pyvista import examples
+    >>> path = examples.download_dicom_stack(load=False)
+    >>> reader = pyvista.DICOMReader(path)
+    >>> dataset = reader.read()
+    >>> dataset.plot(volume=True, zoom=3, show_scalar_bar=False)
+
+    """
+
+    _class_reader = _vtk.vtkDICOMImageReader
+
+    def __init__(self, path: str):
+        """Initialize reader from DICOM file (*.dcm) or folder of DICOM files."""
+        self._reader = self._class_reader()
+        self._progress_bar = False
+        self._progress_msg = None
+        self.directory = None
+
+        if os.path.isfile(path):
+            self.filename = path
```

Comment:
This is already done inside `_set_filename`?

Reply:
Resolved in abcd8aa.